### PR TITLE
44229 consolidate forloebsside with webform content type

### PR DESCRIPTION
**Pull request by madsnorgaard**

### CODE REVIEW

File: src/Plugin/EngineTasks/MaestroSelectContentTask.php [hunk 1]
```diff
@@ -323,35 +323,35 @@ public function handleExecuteSubmit(array &$form, FormStateInterface $form_state
     $processID = MaestroEngine::getProcessIdFromQueueId($queueID);
 
     if (strstr($triggeringElement['#id'], 'edit-submit') !== FALSE && $queueID > 0) {
-      // If existing forløbsside is chosen simply assign it to this process
-      // If a new artivle is chosen create that forløbsside
+      // If existing self-service page is chosen simply assign it to this process
+      // If a new artivle is chosen create that self-service page
```
Comment (lats1) [inline src/Plugin/EngineTasks/MaestroSelectContentTask.php:327]: Typo?

**Review by lats1 — DISMISSED** [review]

**Comment by lats1** [comment]
The commit's in the top, e.g. `Add module uninstall logic...`. Should they be included in this pr?

File: src/Plugin/EngineTasks/MaestroSelectContentTask.php (hunk 1 above)
Comment (madsnorgaard) [reply src/Plugin/EngineTasks/MaestroSelectContentTask.php:327]: I do not see a typo but I had to refactor the forløbsside content type and thus translated "selvbetjeningsside" as "self-service page".

Should we merely use the Danish maching name going forward - "selvbetjeningsside"?

**Comment by madsnorgaard** [comment]
> The commit's in the top, e.g. `Add module uninstall logic...`. Should they be included in this pr?

For now yes. We should should begin looking at moving as many configuration files as possible from the install profile to the module. This should make everything a bit more flexible.

File: src/Plugin/EngineTasks/MaestroSelectContentTask.php (hunk 1 above)
Comment (lats1) [reply src/Plugin/EngineTasks/MaestroSelectContentTask.php:327]: Ah I should have been clearer. `artivle` should probably be something else.

**Comment by lats1** [comment]
Ah I see. Disregard my comment then :)

File: src/Plugin/EngineTasks/MaestroSelectContentTask.php (hunk 1 above)
Comment (lats1) [reply src/Plugin/EngineTasks/MaestroSelectContentTask.php:327]: I'm a fan of using english machine-names.

**Pull request merged**